ui/components/app/snaps/snap-ui-renderer/components/section.ts

```diff
@@ -0,0 +1,25 @@
+import { SectionElement, BoxElement } from '@metamask/snaps-sdk/jsx';
+import {
+  BackgroundColor,
+  BorderRadius,
+} from '../../../../../helpers/constants/design-system';
+import { UIComponentFactory, UIComponentParams } from './types';
+import { box } from './box';
+
+export const section: UIComponentFactory<SectionElement> = ({
+  element,
+  ...params
+}) => {
+  const { children, props } = box({ element, ...params } as unknown as UIComponentParams<BoxElement>);
+  return {
+    element: 'Section',
```

Comment:
Is there a component in the extension called `Section` ? Otherwise we might want to use `Box`

(If there is one, you need to add it to the allowed component list of the template renderer)